lib/route/cls/u32.c

```diff
@@ -553,6 +571,7 @@ struct rtnl_act* rtnl_u32_get_action(struct rtnl_cls *cls)
     if (!(u->cu_mask & U32_ATTR_ACTION))
         return NULL;
 
+    rtnl_act_get(u->cu_act);
```

Comment:
this part should be a separate patch.

Also, it is quite a change in behavior, and will result in leaks for previously correct programs. This cannot be done. Instead, I think you can only document this behavior (of not increasing the ref count).

also, I don't understand the comment "Fix rtnl_u32_get_action() so it no longer does a raw memcpy of the action object.". `rtnl_u32_get_action()` returns the pointer value, why do you talk about raw memcpy here?

Reply:
Alright. I'll remove this line from current patch, as it will definitely result in leaks for existing programs.